arches/app/media/js/views/graph/graph-designer/graph-tree.js

```diff
@@ -85,6 +86,30 @@ define([
             }, this);
         },
 
+        /**
+         * Returns a boolean to indicate whether this node participates in descriptor function
+         * @param {object} node - a node in the tree 
+         */
+        isFuncNode: function(node) {
+            var appFuncDesc, appFuncName;
+            if(this.appliedFunctions().length > 0) {
+                appFuncDesc = this.appliedFunctions()[0]['config']['description']['nodegroup_id'];
```

Comment:
this section has the same problems as described above re: isFuncNode in card.js, ie keying on properties that may not be present and assuming the function of interest is at index 0 of the array.